cpp/BUILD.bazel

```diff
@@ -149,7 +150,7 @@ cc_test(
         "src/ray/test/cluster/*.cc",
     ]),
     args = [
-        "--ray-dynamic-library-path=$(location cluster_mode_test.so)",
+        "--ray-code-search-path=$(location cluster_mode_test.so)",
```

Comment:
Do we still need this argument for tests?

Reply:
Yes, we need to make sure this argument works. And the argument of example test has been removed. https://github.com/ray-project/ray/pull/16828/files#diff-1706c14d71810b14bd8b98c70f7651d2ea3f7f50225d2e2d352759251ce24ab5L7